.eslintrc.json

```diff
@@ -40,28 +50,57 @@
       }
     ],
     "keyword-spacing": "error",
+    "linebreak-style": "error",
+    "max-statements-per-line": "error",
```

Comment:
On closer inspection, the rule (which has `max=1` by default) does not complain about this in `htmlparser.js`:
```
if (args[3] === '') { delete args[3]; }
if (args[4] === '') { delete args[4]; }
if (args[5] === '') { delete args[5]; }
```

which according to [their documentation](http://eslint.org/docs/rules/max-statements-per-line#max) is clearly incorrect.

I'll go with your suggestion and remove this option then.